### PR TITLE
Compile css for icons style

### PR DESCRIPTION
**Pull request by bjarnef**
### Prerequisites

- [x] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes https://github.com/umbraco/Umbraco-CMS/issues/14614

### Description
When rendering icons inside `<umb-block-grid-root>` and `<umb-block-grid-block>` it only rendered SVG icons and not legacy font icons like `icon-science` which doesn't have a SVG version here: https://github.com/umbraco/Umbraco-CMS/tree/contrib/src/Umbraco.Web.UI.Client/src/assets/icons

We should probably also add `icon-science.svg`, but font icons are still supported and developers may still use custom font icons (font packages) although SVG icons are the modern approach.

The PR also remove some redundant style if we sometime later need to adjust style of `<umb-icon>` component, this ensure icons in block grid are consistent. 

In the block grid view we probably need to adjust the CSS to include cache buster  like here:

https://github.com/umbraco/Umbraco-CMS/blob/71d990504ee583204b76f45e7bf6d336b5e04882/src/Umbraco.Web.UI.Client/src/views/propertyeditors/blockgrid/umbblockgridroot.component.js#L39

image

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/less/belle.less [hunk 1]
```diff
@@ -73,7 +73,8 @@
 @import "modals.less";
 @import "panel.less";
 @import "sections.less";
-@import "helveticons.less";
+//@import "helveticons.less";
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI.Client/src/less/belle.less:76]:
```suggestion
```
No need to keep commented out code, that's what Giit history is for.

File: src/Umbraco.Web.UI.Client/src/less/belle.less [hunk 2]
```diff
@@ -150,7 +151,7 @@
 @import "components/umb-color-swatches.less";
 @import "components/check-circle.less";
 @import "components/umb-file-icon.less";
-@import "components/umb-icon.less";
+//@import "components/umb-icon.less";
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI.Client/src/less/belle.less:154]:
```suggestion
```

No need to keep commented out code, that's what Giit history is for.

**Review by nul800sebastiaan — CHANGES_REQUESTED** [review]
Looking good @bjarnef, 2 comments to resolve. 

Other than that you noted:

> In the block grid view we probably need to adjust the CSS to include cache buster 

Please do yes!

File: src/Umbraco.Web.UI.Client/src/less/belle.less (hunk 1 above)
Comment (bjarnef) [reply src/Umbraco.Web.UI.Client/src/less/belle.less:76]: Yes, sure 🙂

**Comment by nul800sebastiaan** [comment]
Thanks @bjarnef this is looking good now, if you still want to adjust the cachebuster then I look forward to a new PR for it! 👍

**Comment by bjarnef** [comment]
@nul800sebastiaan the cache buster is adjusted in https://github.com/umbraco/Umbraco-CMS/pull/14621

**Pull request merged**